lib/fog/aws/requests/rds/list_tags_for_resource.rb

```diff
@@ -11,27 +11,40 @@ class Real
         # ==== Returns
         # * response<~Excon::Response>:
         #   * body<~Hash>:
+
         def list_tags_for_resource(rds_id)
+          resource_name = "arn:aws:rds:#{@region}:#{owner_id}:db:#{rds_id}"
+          %w[us-gov-west-1 us-gov-east-1].include?(@region) ? resource_name.insert(7, '-us-gov') : resource_name
           request(
-            'Action'        => 'ListTagsForResource',
-            'ResourceName'  => "arn:aws:rds:#{@region}:#{owner_id}:db:#{rds_id}",
-            :parser         => Fog::Parsers::AWS::RDS::TagListParser.new
+            'Action' => 'ListTagsForResource',
+            'ResourceName' => resource_name,
+            :parser => Fog::Parsers::AWS::RDS::TagListParser.new
           )
         end
+
+        def list_tags_for_snapshot(snapshot_name)
```

Comment:
Similarly here, I think we might want to pull this out to something like: `lib/fog/aws/requests/rds/list_tags_for_snapshot.rb` and add tests and mock functionality.

Reply:
@geemus Yes
 removed rds snapshot changes will add in another PR.